script.exp

```diff
@@ -0,0 +1,150 @@
+#!/usr/local/bin/expect -f
```

Comment:
It looks like this file was not intended to be added so source control.

Reply:
Whoops.